diff --git a/packages/scratch-gui/src/index-standalone.tsx b/packages/scratch-gui/src/index-standalone.tsx
index 317472af7..de830bf65 100644
--- a/packages/scratch-gui/src/index-standalone.tsx
+++ b/packages/scratch-gui/src/index-standalone.tsx
@@ -6,8 +6,8 @@ import {EditorState} from './lib/editor-state';
 import {ReactComponentLike} from 'prop-types';
 import {compose} from 'redux';
 
-export {EditorState, EditorStateParams} from './lib/editor-state';
-export {AccountMenuOptions} from './lib/account-menu-options';
+export {EditorState, type EditorStateParams} from './lib/editor-state';
+export {type AccountMenuOptions} from './lib/account-menu-options';
 
 export {setAppElement} from 'react-modal';
 
diff --git a/packages/scratch-gui/src/lib/save-project-to-server.js b/packages/scratch-gui/src/lib/save-project-to-server.js
index 39b8b06ba..39b47b391 100644
--- a/packages/scratch-gui/src/lib/save-project-to-server.js
+++ b/packages/scratch-gui/src/lib/save-project-to-server.js
@@ -4,7 +4,7 @@ import xhr from 'xhr';
 /**
  * Save a project JSON to the project server.
  * This should eventually live in scratch-www.
- * @param {string} projectHost the hostname of the project service.
+ * @param {string|undefined} projectHost the hostname of the project service.
  * @param {number} projectId the ID of the project, null if a new project.
  * @param {object} vmState the JSON project representation.
  * @param {object} params the request params.
